content/blog/2025-11-07-tmil-22/index.md

```diff
@@ -0,0 +1,128 @@
++++
+title = "Linebender in October 2025"
+authors = ["Raph Levien"]
++++
+Linebender is an informal open-source organization working on various projects to advance the state of the art in GUI for [the Rust programming language](https://rust-lang.org).
+
+October saw steady progress on our core crates, including a number of releases.
+
+## Vello
+
+Vello is our GPU vector renderer.
+It can draw large 2D scenes with high performance, using GPU compute shaders for most of the work.
+
+Laurenz Stampl has done his Masters degree at ETH Zurich on high performance CPU rendering of 2D graphics, particularly the techniques used in vello_cpu.
+The [masters thesis] is now published.
+It contains up-to-date background on 2D rendering, which is not well represented in the literature, as well as an explanation of the sparse strip method, and benchmarks.
+
+<figure>
+
+<img style="height: auto" width="1170" height="920" src="sparse_strip.png" alt="The tiling of a butterfly shape at four different levels of granularity. The top left shows many small tiles, while the bottom right shows fewer large ones, but also a much smaller fraction of space not covered by a tile.">
+
+<!-- <img style="height: auto; width: 50%; margin-left: 25%;" width="666" height="673" src="hybrid-blending.png" alt="A series of overlapping shapes of various colours, variously composited."> -->
+
+<figcaption>
+
+An illustration from Laurenz Stampfl's Masters Thesis, showing the effect of tile size on rendering efficiency.
+
+</figcaption>
+</figure>
+
+We released version [0.0.4 of the sparse strips][sparse strips 0.0.4] versions of the renderer.
+It should be a solid version of vello_cpu, suitable for a wider range of applications where CPU-only rendering is appropriate.
```

Comment:
```suggestion
It should be a solid version of Vello CPU, suitable for a wider range of applications where CPU-only rendering is appropriate.
```

We could also consider linking to Vello CPU on crates.io here, although I'm more hesitant to do the same for Vello Hybrid...

```suggestion
It should be a solid version of [Vello CPU](https://crates.io/crates/vello_cpu), suitable for a wider range of applications where CPU-only rendering is appropriate.
```